### DISCUSSION TITLE
What kinds of things can I do with GitHub apps

### DISCUSSION THREAD

**Discussion by ghfan** [community/community#24905] [Apps, API and Webhooks]
<p>Other than adding labels to issues, as shown in the quickstart guide, what sorts of things can I do with GitHub apps?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/ghfan">@ghfan</a> Thanks for being part of the GitHub Community Forum!</p>
<p>GitHub Apps are the most powerful way to utilize GitHub’s expansive REST and GraphQL APIs. In addition to having access to <a href="https://developer.github.com/v3/apps/available-endpoints/" rel="nofollow noopener">most of the API endpoints</a> available to developers, only GitHub Apps have access to the <a href="https://developer.github.com/v3/checks/" rel="nofollow noopener">Checks API</a>. GitHub Apps also have granular permissions that allow apps access to only what they need, which is an advantage over OAuth apps. For examples of smaller apps, you can check out <a href="https://probot.github.io/apps" rel="nofollow noopener">Probot apps</a>. For examples of larger scale apps, check out <a href="https://github.com/marketplace" rel="nofollow noopener">GitHub Marketplace</a>!</p>
<p>I hope this helps!</p>
